great_expectations/dataset/pandas_dataset.py

```diff
@@ -520,12 +520,24 @@ def expect_column_values_to_be_of_type(self, column, type_,
             "float": [float, np.float_],
             "double": [float, np.longdouble],
             "bytes": [bytes, np.bytes_],
-            "string": [string_types, np.string_]
+            "string": [np.string_] + list (string_types)
```

Comment:
Yikes. How did this get past us before?

Reply:
Before, the code always called `tuple()` on the list as part of the lambda, and use `isinstance`. Now, in the event that the whole column is of a specific dtype we bypass that check entirely and rely on `in` which requires the list and doesn't re-call tuple.
![image](https://user-images.githubusercontent.com/332575/56756340-93c18c80-675f-11e9-82fa-d7058ac62954.png)